assets/prototype/domain/eventEditor/context/DateTimeProvider/index.tsx

```diff
@@ -9,15 +9,16 @@ import {
 	EditorState,
 } from '../../../../application/ui/components/layout/editor-modal/useEditorModalState/types';
 
-export interface DateTimeContextProps extends ContextProps {}
-export const DateTimeContext = createContext({} as DateTimeContextProps);
+export type DateTimeContextProps = ContextProps;
+export const DateTimeContext = createContext<DateTimeContextProps>(null);
 
 interface DatetimeProviderProps {
-	children: React.ReactChildren;
+	children?: React.ReactNode;
```

Comment:
Why this needs to be optional?

Reply:
Otherwise you will have to pass `children` as prop 😄